dataRetriever/resolvers/trieNodeResolver.go

```diff
@@ -115,14 +115,14 @@ func (tnRes *TrieNodeResolver) resolveOnlyRequestedHashes(hashes [][]byte, nodes
 			continue
 		}
 
+		spaceUsed += len(serializedNode)
+		nodes[string(serializedNode)] = struct{}{}
+		remainingSpace -= len(serializedNode)
+
```

Comment:
on line 133 there is a "seriali**a**zedNodes" -> serializedNodes

Reply:
fixed

Reply:
ok